pandas/core/describe.py

```diff
@@ -337,6 +337,8 @@ def describe_timestamp_1d(data: Series, percentiles: Sequence[float]) -> Series:
     # GH-30164
     from pandas import Series
 
+    from pandas.io.formats.format import format_percentiles
```

Comment:
I assume these two changes weren't intended?

Reply:
They are necessary to resolve a circular import problem

Reply:
Might be good to in the future re-organize the doc-manipulations into one file that has no other pandas imports. No need to address this in this PR.

Reply:
umm can you show exactly what he problem is? pandas.io.formats.format is designed to be importable here